packages/lorcana-cards/src/cards/002/characters/149-hiram-flaversham-toymaker.ts

```diff
@@ -0,0 +1,45 @@
+import type { CharacterCard } from "@tcg/lorcana";
+
+export const hiramFlavershamToymaker: CharacterCard = {
+  id: "slt",
+  cardType: "character",
+  name: "Hiram Flaversham",
+  version: "Toymaker",
+  fullName: "Hiram Flaversham - Toymaker",
+  inkType: ["sapphire"],
+  franchise: "Great Mouse Detective",
+  set: "002",
+  text: "ARTIFICER When you play this character and whenever he quests, you may banish one of your items to draw 2 cards.",
+  cost: 4,
+  strength: 1,
+  willpower: 6,
+  lore: 1,
+  cardNumber: 149,
+  inkable: true,
+  externalIds: {
+    ravensburger: "671965c7095dd8a31d791f102a4dc3e789f6a21b",
+  },
+  abilities: [
+    {
+      id: "slt-1",
+      text: "ARTIFICER When you play this character and whenever he quests, you may banish one of your items to draw 2 cards.",
+      name: "ARTIFICER",
+      type: "triggered",
+      trigger: {
```

Comment:
In this case we have two triggers: When you play this character and whenever he quests. think about a way of addressing it.

The effect also has a cost of banishing one of your items.